website_sale_payment_term/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,4 @@
+Configuration
+=============
+
+No configuration can take place
```

Comment:
Only this line needed. Just say: no specific configuration is needed for this module.